arch/arm64/src/common/arm64_gicv3.c

```diff
@@ -942,7 +942,7 @@ static void arm64_gic_init(void)
 
   cpu             = this_cpu();
```

Comment:
shoud be `up_cpu_index()`

Reply:
In AMP and up_cpu_index() may return the index of the physical core.  gic_rdists[cpu] may be overflow in next line, but this_cpu alway 0 in that case.